documentation/modules/auxiliary/scanner/misc/rocketmq_version.md

```diff
@@ -0,0 +1,41 @@
+## Vulnerable Application
+
+Version scanner for the Apache RocketMQ product.
+
+### Setup
+
+Instructions taken from https://github.com/Malayke/CVE-2023-33246_RocketMQ_RCE_EXPLOIT
+
+```
+docker pull apache/rocketmq:4.9.4
+# Start nameserver
+docker run -d --name rmqnamesrv -p 9876:9876 apache/rocketmq:4.9.4 sh mqnamesrv
```

Comment:
```suggestion
docker run --rm --name rmqnamesrv -p 9876:9876 apache/rocketmq:4.9.4 sh mqnamesrv
```

I might be the odd one out here - but I'm not a fan of throw-away docker instances being run as a daemon, and not removing after themselves by default. Will wait for other folk to chime in though 😄